### ISSUE TITLE
Number of train files vs length of file list

### ISSUE THREAD

**Issue by dreamk73** [CSTR-Edinburgh/merlin#84]
If I have a certain file list as input and I change the number of train, validation, test files to smaller numbers, I would expect merlin to only take those number of files from the file list, but it seems to still run all label files through the normalizer. Is that expected behavior? Should one adapt the file list to match train + valid + test?

**Comment by bajibabu** [comment]
Yes, it is expected, but the min-max or mean-variance values are computed on only training data. That is a good idea to adapt the file list to match train+valid +test.

**Comment by dreamk73** [comment]
I think it would be a good idea. I set up a structure where I keep my file list and label files and acoustic feature files in a different location so I can retrain with different setups without duplicating a lot of data. If I change the number of train files down to test if something works, I don't necessarily want to have to change the file list.
